cfssl/CHANGELOG.md

```diff
@@ -0,0 +1,2 @@
+# CHANGELOG - cfssl
+
```

Comment:
Can you add a changelog entry, something like: https://github.com/DataDog/integrations-extras/blob/master/apptrail/CHANGELOG.md